<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_testXml</class>
 <widget class="QDialog" name="Dlg_testXml">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="button_Selectionner">
         <property name="text">
          <string>Selectionner</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Fermer">
         <property name="text">
          <string>&amp;Fermer</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_Fermer</sender>
   <signal>clicked()</signal>
   <receiver>Dlg_testXml</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
